src/Razor/test/Microsoft.NET.Sdk.Razor.Test/IntegrationTests/PublishIntegrationTest.cs

```diff
@@ -60,6 +60,53 @@ public async Task Publish_PublishesAssembly()
             Assert.FileCountEquals(result, 0, Path.Combine(PublishOutputPath, "Views"), "*.cshtml");
         }
 
+        [Fact]
+        [InitializeTestProject("SimpleMvcWithContent")]
+        public async Task Publish_PublishesContent()
```

Comment:
Do we need a whole new test or test project for this?

Reply:
I could modify the current ones but I didn't want to just add assertions to all of them since they seem to be verifying other specific things.

Reply:
I prefer if this is a different test.

Reply:
Why is that the best choice? each of these tests take 4-5 seconds to run. 

Don't add another project if we don't need it.

Reply:
If it's effectively the same set of files, just re-use the project.

Reply:
What’s the concrete suggestion? Add these files to the existing mvc project and add the assert to all of the tests that use it?

Reply:
Nah, I'll just put it in one of them.